channels/pkg/cmd/apply_channel.go

```diff
@@ -46,8 +46,10 @@ func NewCmdApplyChannel(f Factory, out io.Writer) *cobra.Command {
 	var options ApplyChannelOptions
 
 	cmd := &cobra.Command{
-		Use:   "channel",
-		Short: "Apply channel",
+		Use:     "channel CHANNEL",
```

Comment:
Changes to channels should not be needed anymore.

Reply:
Yeah, but it's still more correct now :p I think we should keep this change.

Reply:
Maybe just remove the example, or update the command?